### ISSUE TITLE
[BUG] War Room managemnt pop up details are not accurate

### ISSUE THREAD

**Issue by ivangergo3** [netdata/netdata-cloud#383]
**Describe the bug**	

The number of Nodes and users is not accurate for War Room Management

**To Reproduce**	

Prerequisite: Have a user with multiple war rooms

Steps:
1. Open an incognito window
2. Log into netdata
3. Go into one of your rooms
4. Open Manage War rooms popup

Observe, that all rooms will have 0 nodes and 0 users, except the one where you are right now.
If you open the manage popup for all rooms, the data will be valid for all of them.

**Expected behaviour**	

See the actual number of users and nodes in all War Room.	

**Screenshots**	

![Capture](https://user-images.githubusercontent.com/5202868/161540229-9abca03a-c23d-4e67-a66c-2332e710eee0.JPG)

**Error logs**	
1. Open developer devtools (e.g for [chrome](https://developers.google.com/web/tools/chrome-devtools/open)	
2. Submit a screenshot or better a [HAR file](https://support.zendesk.com/hc/en-us/articles/204410413-Generating-a-HAR-file-for-troubleshooting)	

**Desktop (please complete the following information):**	
 - OS: Windows 10
 - Browser: Chrome
 - Version: Version 100.0.4896.60 (Official Build) (64-bit)	

**Additional context**

**Comment by dionysis-netdata** [comment]
I suspect this is superseded by https://github.com/netdata/netdata-cloud/issues/281 GOAL Q2

Admins should see a list of **all** war rooms in a space (public private) and statistics to be accurate

**Comment by ravilushqa** [comment]
also I've found the mechanics of the bug. if we jump through all the rooms and click manage space -> war rooms
we will see actual state. After that we can refresh the page, and we will see actual state only for current room, and 0 for others

**Comment by ivangergo3** [comment]
@dionysis-netdata you might be right
Let's wait for that feature, and we can come back to this one and check it again.
We definitely have to check it at some point

@ravilushqa I think it is basically the same problem, so we do not see all the details. I guess the frontend asking only for the current status. If we will have the new feature it can sort this issue. But let's come back after the feature is ready

**Comment by ravilushqa** [comment]
@AlexNti expected output from `/api/v1/spaces/{spaceID}/rooms` will be:

**no nodes and members**
```json
{
  "updatedAt": "2022-06-24T15:34:15.213943+04:00",
  "results": [
    {
      "id": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "slug": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "name": "",
      "description": "",
      "isMember": false,
      "untouchable": false,
      "default": false,
      "private": false,
      "state": "",
      "createdAt": "0001-01-01T00:00:00Z",
      "node_ids": [],
      "member_ids": []
    }
  ]
}
```

**nodes and members exists**
```json
{
  "updatedAt": "2022-06-24T15:34:15.213943+04:00",
  "results": [
    {
      "id": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "slug": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "name": "",
      "description": "",
      "isMember": false,
      "untouchable": false,
      "default": false,
      "private": false,
      "state": "",
      "createdAt": "0001-01-01T00:00:00Z",
      "node_ids": [
        "4c0d2ec2-675f-469a-80b6-829350492fd0",
        "6aca085a-504b-494e-b5a8-f543c27952a7"
      ],
      "member_ids": [
        "eefc0c57-1885-4169-9ef0-7c4fde0d8bd5",
        "bed507cb-3c39-4be0-8497-ff1438997c22"
      ]
    }
  ]
}{
  "updatedAt": "2022-06-24T15:34:15.213943+04:00",
  "results": [
    {
      "id": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "slug": "6ee91d04-63c3-4cd9-8677-11bbb69c6096",
      "name": "",
      "description": "",
      "isMember": false,
      "untouchable": false,
      "default": false,
      "private": false,
      "state": "",
      "createdAt": "0001-01-01T00:00:00Z",
      "node_ids": [
        "4c0d2ec2-675f-469a-80b6-829350492fd0",
        "6aca085a-504b-494e-b5a8-f543c27952a7"
      ],
      "member_ids": [
        "eefc0c57-1885-4169-9ef0-7c4fde0d8bd5",
        "bed507cb-3c39-4be0-8497-ff1438997c22"
      ]
    }
  ]
}
```
